qemu/tests/cfg/win_guest_debugging_tool.cfg

```diff
@@ -13,10 +13,15 @@
     cdrom_virtio = isos/windows/virtio-win.iso
     cmd_findstr_in_file = type %s | findstr "%s"
     include_sensitive_data = False
+    script_execution_timeout = 720
     target_files = "msinfo32.txt,system.evtx,security.evtx,application.evtx,drv_list.csv,virtio_disk.txt,WindowsUpdate.log,Services.csv,WindowsUptime.txt,RunningProcesses.csv,InstalledApplications.csv,InstalledKBs.csv,NetworkInterfaces.txt,IPConfiguration.txt,setupapi.dev.log,setupapi.setup.log,setupapi.offline.log,ErrorWindowsUpdate.log,OutputWindowsUpdate.log,LocaleMetaData"
     target_dump_files = "MEMORY.DMP,Minidump"
     script_name = "CollectSystemInfo.ps1"
     cmd_search_file_global = powershell.exe -Command "Get-PSDrive -PSProvider FileSystem | ForEach-Object { Get-ChildItem -Path $_.Root -Recurse -Filter '%s' -ErrorAction SilentlyContinue } | ForEach-Object { Join-Path -Path $_.Directory.FullName -ChildPath $_.Name }"
+    Win2016:
+        script_execution_timeout = 1200
+        cmd_disable_ie_esc_admin = powershell.exe -Command "Set-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Active Setup\Installed Components\{A509B1A7-37EF-4b3f-8CFC-4F3A74704073}' -Name 'IsInstalled' -Value 0"
+        cmd_disable_ie_esc_user = powershell.exe -Command "Set-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Active Setup\Installed Components\{A509B1A8-37EF-4b3f-8CFC-4F3A74704073}' -Name 'IsInstalled' -Value 0"
```

Comment:
As we dicsussed offline, it would be better to break down current commit, it would be more clear.

Reply:
Yeah, I'll split another one, as for this one, just keep it here this time. thanks